client/agent.go

```diff
@@ -300,6 +301,8 @@ func (c *FuzzitClient) runLibFuzzerRegression() error {
 		"-exact_artifact_path=./artifact",
 		"-error_exitcode=76",
 	}, regressionFiles...)
+	args = append(args, strings.Split(c.currentJob.Args, " ")...)
```

Comment:
same comment as above but regarding the regressionFiles

Reply:
And here as well